Game1.cs

```diff
@@ -67,8 +71,8 @@ protected override void Update(GameTime gameTime)
             if (!battleSystem.ongoingBattle)
             {
                 battleSystem.InitializeFight(new List<Actor> {
-                    new Actor("player", new Vector2(100, 100), 100, GraphicsDevice),
-                    new Actor("Enemy", new Vector2(600, 100), 100, GraphicsDevice)
+                    new Actor("Player", new Vector2(100, 100), 100, new List<IAction> { new ActionAttack() , new ActionHeal(), new ActionNothing() }, GraphicsDevice, new PlayerTurnBehaviour()),
+                    new Actor("Enemy", new Vector2(600, 100), 100, new List<IAction> { new ActionAttack() , new ActionNothing() }, GraphicsDevice, new RandomEnemyTurnBehaviour())
```

Comment:
again, Actors should not get a GraphicsDevice, them getting a texture is probably better.